src/test/java/io/strimzi/kafka/metrics/KafkaMetricsCollectorTest.java

```diff
@@ -108,6 +109,19 @@ public void testCollectNonNumericMetric() {
         assertEquals(expectedLabels, snapshot.getDataPoints().get(0).getLabels());
     }
 
+    @Test
+    public void testLabelNameSanitizing() {
```

Comment:
We want to avoid tests that only exercise code from dependencies. This test only validates `PrometheusNaming.sanitizeLabelName()` and it does not check anything from our metric reporter.

Reply:
Removed